src/diffusers/loaders.py

```diff
@@ -1249,12 +1337,20 @@ def _convert_kohya_lora_to_diffusers(cls, state_dict):
                     diffusers_name = diffusers_name.replace("to.k.lora", "to_k_lora")
                     diffusers_name = diffusers_name.replace("to.v.lora", "to_v_lora")
                     diffusers_name = diffusers_name.replace("to.out.0.lora", "to_out_lora")
+                    diffusers_name = diffusers_name.replace("proj.in", "proj_in")
+                    diffusers_name = diffusers_name.replace("proj.out", "proj_out")
```

Comment:
Will these values be always present?

Reply:
They might not but this is a precautionary replace so if they are not found it won't do anything. Here are a few examples though where they are present:
```
217:lora_unet_down_blocks_0_attentions_0_proj_in.alpha
218:lora_unet_down_blocks_0_attentions_0_proj_in.lora_down.weight
433:lora_unet_mid_block_attentions_0_proj_in.alpha
434:lora_unet_mid_block_attentions_0_proj_in.lora_down.weight
469:lora_unet_up_blocks_1_attentions_0_proj_in.alpha
759:lora_unet_up_blocks_3_attentions_2_proj_in.lora_up.weight
```